From ca9a65cea92497900605e31299beefc6626368b1 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 30 Aug 2017 15:23:56 +0800
Subject: [PATCH] clk: rockchip: rk3128: add pclk_pmu as critical clock

pclk_pmu need always on, and no dts node to handle this clk,
so make it as critical clock

Change-Id: If95d7c45f8935883a335846b42f526cd2f36f131
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3128.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3128.c b/drivers/clk/rockchip/clk-rk3128.c
index 90b688b4a57a..fb8cc6248efe 100644
--- a/drivers/clk/rockchip/clk-rk3128.c
+++ b/drivers/clk/rockchip/clk-rk3128.c
@@ -577,6 +577,7 @@ static const char *const rk3128_critical_clocks[] __initconst = {
 	"aclk_peri",
 	"hclk_peri",
 	"pclk_peri",
+	"pclk_pmu",
 };
 
 static struct rockchip_clk_provider *__init rk3128_common_clk_init(struct device_node *np)
-- 
2.35.3

